src/compat/responder.js

```diff
@@ -4,6 +4,7 @@ const OS = require('os')
 const MDNS = require('multicast-dns')
 const log = require('debug')('libp2p:mdns:compat:responder')
 const { SERVICE_TAG_LOCAL } = require('./constants')
+const { Buffer } = require('buffer')
```

Comment:
Is there a reason for including `buffer`? mdns only works in Node, so Buffer should always be available.